### ISSUE TITLE
Need to improve user experience

### ISSUE THREAD

**Issue by sahu-iisc** [CodefosterGS/cf_frontend#4]
As our website is already live and people are visiting it, therefore I think we need to improve the UI/UX.
There are some problems with the UI/UX like lack of animations, proper images, content, loader etc. Also there is a problem with navbar and the website lacks some features also.
So in my opinion we should figure out all this problems as soon as possible @ashish3805 @Siddhant021295 @f71uday

**Comment by ashish3805** [comment]
I agree that we lack good graphics. Also, the site should be more vibrant. @Himanshu-Sahu Can you compile a list of changes or tasks and create individual issues for them. Bunch them in To-Do's at this project https://github.com/CodefosterGS/cf_frontend/projects/2

**Comment by ashish3805** [comment]
@f71uday @Siddhant021295  Can you get someone from codefoster to help with graphics and UI/UX?
